package-lock.json

```diff
@@ -5028,9 +5028,8 @@
 			}
 		},
 		"@rocket.chat/apps-engine": {
-			"version": "1.18.0",
-			"resolved": "https://registry.npmjs.org/@rocket.chat/apps-engine/-/apps-engine-1.18.0.tgz",
-			"integrity": "sha512-Y9XgRnG8v4HtujaMXuzTd9hjycQX9+wDOE30HhHL1NQl7de4+9MzPdWDetdy+84cBrDtXCymVYF/TzmLf+69Mw==",
+			"version": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#e1a0df0c200b2039c1866e2856f8d86f8c431da6",
```

Comment:
Thanks @Shailesh351........  but this seems so brittle.  We will have to constantly keep track of the most recent hash and make sure we remember to update it any time we make changes to the apps-engine. 

For now we will have to use this, and I'll verify that it works... but how long do you think it will be that we will have to be using our apps-engine fork?  Do you think we will be making more changes?

Thanks.

Reply:
We will not be making any new changes in the apps-engine directly. But whenever we do upstream catchup we need to update package-lock file. This file is automatically updated when we do `meteor npm install`.

or 
we can remove using cached node_modules in CICD so that it will get updated node_modules for every build.